awx/main/models/events.py

```diff
@@ -57,7 +58,18 @@ def create_host_status_counts(event_data):
     return dict(host_status_counts)
 
 
+MINIMAL_EVENTS = set([
+    'playbook_on_play_start', 'playbook_on_task_start',
+    'playbook_on_stats', 'EOF'
+])
+
+
 def emit_event_detail(event):
+    if (
+        settings.UI_LIVE_UPDATES_ENABLED is False and
```

Comment:
It seems not unlikely that a customer might want the reduced broadcast behavior for job_events without having to turn off live updates for job lists. I'm wondering if we should this it's own setting outside of live updates so that the ui can handle them independently.

cc @ryanpetrello - thoughts?

Reply:
Maybe so?  That said, the notion of asking a user to properly understand and toggle two things independently concerns me a little 😄.  One of my goals with this PR is for `UI_LIVE_UPDATES_ENABLED` to effectively serve as a one-click "low bandwidth mode" for users who want higher event throughput.

Reply:
:thumbsup: sgtm

Reply:
UI_LIVE_UPDATES_ENABLED was originally designed for the old list update scheme (where websocket triggered an API refresh).

I'd be more tempted to tweak it so it means what this PR does instead and still allow list refreshes with it.

Reply:
Yep - You both have a point and I'm convinced. sgtm!

Reply:
Note: that means we may want to *remove* the bit where UI_LIVE_UPDATES_ENABLED stops list updates. But that can be a second PR.

Reply:
Maybe so.  I suppose you could argue that at this point, we've probably mostly resolved the original problem that `UI_LIVE_UPDATES_ENABLED` was intended to paper over.

Reply:
> Maybe so. I suppose you could argue that at this point, we've probably mostly resolved the original problem that UI_LIVE_UPDATES_ENABLED was originally meant to paper over.

Yea that _should_ be the case at this point

Reply:
Ok - just chatted w/ @mabashian about this. I'm not entirely sure what the scope of work would be off the top of my head but from what we can see it may not be a risky change & something we can fit in. I'll take a closer look tomorrow.

Reply:
Just to be clear, let's aim for this as a separate `devel` PR, though.

Reply:
k, sounds good. We'll track that work in this issue: https://github.com/ansible/awx/issues/8211